### ISSUE TITLE
extra space in ref links

### ISSUE THREAD

**Issue by kacinash** [CDRH/earlywashingtondc#212]
When case narratives/summaries have a link to a case, something is giving it an extra space at the end of the link.

See, [for example](http://earlywashingtondc.org/cases/oscys.caseid.0251):

![screen shot 2017-04-26 at 1 15 45 pm](https://cloud.githubusercontent.com/assets/6652951/25449787/7d9b509c-2a82-11e7-8d0d-8334a6d4bab1.png)

This is what the code looks like in the document:
`<hi rend="italic"><ref target="oscys.caseid.0011">Mima Queen v. Hepburn</ref></hi>.`

**Comment by jduss4** [comment]
In looking into this, I realized that things aren't being italicized, despite being marked that way in the TEI, so I went ahead and added a little CSS to accomplish that.  https://github.com/CDRH/earlywashingtondc/commit/3e58f8e42408854f0382bf7bec812053f6cfc933

@kacinash, @karindalziel came up with a fix in a branch which will require regenerating all the HTML files.  When you are back, let me know when would be a good time to try it out on dev!  We could run it for that single file to look at there and see if it fixes it before running it on all dev.
